repositories/source.go

```diff
@@ -0,0 +1,75 @@
+package repositories
+
+import (
+	"encoding/csv"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"go-capstone/common"
+	"go-capstone/entities"
+)
+
+func GetEmployee(ID int64) (entities.User, common.AppError) {
+	csvFile, csvOpenerErr := os.Open("./app/mock-data/users.csv")
```

Comment:
If you want to improve this, you could use a config file and put all the routes there